javalib/src/main/scala/java/lang/process/BsdOsSpecific.scala

```diff
@@ -30,28 +31,64 @@ object BsdOsSpecific {
 
   // Beware: BSD layouts other than macOS & FreeBSD have not been tested.
 
-// format: off
-
+  // scalafmt: { align.preset = more }
   type kevent = CStruct6[
-    Ptr[CUnsignedInt],	// ident 	/* identifier for this event */
-    CShort, 		// filter 	/* filter for event */
-    CUnsignedShort, 	// flags 	/* action flags for kqueue */
-    CUnsignedInt, 	// fflags 	/* filter flag value */
-    Ptr[CInt],	 	// data		/* filter data value */
-    Ptr[Byte] 		// void *udata 	/* opaque user data identifier */
-    ]
-
-  type kevent64_s = CStruct7[
-    CUnsignedLongInt, 	// ident 	/* identifier for this event */
-    CShort, 		// filter 	/* filter for event */
-    CUnsignedShort, 	// flags 	/* action flags for kqueue */
-    CUnsignedInt, 	// fflags 	/* filter flag value */
-    CLongInt, 	        // data 	/* filter data value */
-    CUnsignedLongInt,   // udata 	/* opaque user data identifier */
-    Ptr[CUnsignedLongInt] // ext[2]	/* filter-specific extensions */
+    USize,          // ident       /* identifier for this event: uintptr_t */
+    CShort,         // filter      /* filter for event */
+    CUnsignedShort, // flags       /* action flags for kqueue */
+    CUnsignedInt,   // fflags      /* filter flag value */
+    Size,           // data        /* filter data value: intptr_t */
+    Ptr[Byte]       // void *udata /* opaque user data identifier */
```

Comment:
SN `/nativelib/src/main/scala/scala/scalanative/unsafe/package.scala` now has a `CVoidPtr` and
that contemporary practice is to use it rather than the older, then necessary idiom.

This is new code, so I mention mention it for your convenience. I am not suggesting that other existing usages in the same file be tracked down and altered
as much as that would be an improvement in consistency.

Reply:
done, in some sense. replaced with USize.

Reply:
undone, back to Ptr[Byte]. CVoidPtr is Ptr[_] so... equivalent.